roles/core/repositories_client/tasks/centos_8.yml

```
@@ -13,42 +22,11 @@
     option: enabled
     value: '0'
     no_extra_spaces: yes
-  when: ( 'os' in repositories ) or ( item.name is defined and item.name == 'os' )
+  when:
+    - ( 'os' in repositories )
```

Same condition as centos_7?
```suggestion
    - ( 'os' in repositories ) or ( item.name is defined and item.name == 'os' )
```

---

True. Previous code was buggy.

---

#288